jsx/Panel.js

```diff
@@ -64,7 +64,7 @@ class Panel extends Component {
         data-target={'#' + this.props.id}
         style={{cursor: 'pointer', height: '3em'}}
       >
-        {this.props.title}
+        <b>{this.props.title}</b>
```

Comment:
this should be done with CSS, not `<b>`. (There's already a style attribute on the div, you just need to add font-weight to it..)